spec/lib/mcb/commands/courses/show_spec.rb

```diff
@@ -18,8 +18,8 @@ def execute_show(arguments: [], input: [])
     new_provider
   end
 
-  let(:subject1) { create(:subject, :primary_with_mathematics) }
-  let(:subject2) { create(:subject, :primary_with_mathematics) }
+  let(:subject1) { create(:primary_subject, :primary_with_mathematics) }
+  let(:subject2) { create(:primary_subject, :primary_with_mathematics) }
```

Comment:
Because we are creating subjects in `spec/support/reference_data.rb`, pretty much all subject fixtures should be `find_or_create`

Reply:
btw, I'd recommend running tests after just changing a few of these, there's always the possibility that it'll break things, even though it's the way things should be done. If you find that you can't use `find_or_create` in a specific test, then might want to use the tag `without_subjects: true` on a test to make sure that the subjects table is empty before it runs (see the `reference_data.rb` file for how that is used.